File: uri/bonus/bonus.scala
object Main{
    def calculabonus(salario:Double, montante:Double):Double =
        salario + ( (montante * 15.0) / 100.0 )

    def main(args:Array[String])
    {
        var nome = scala.io.StdIn.readLine() : String
        var salario = scala.io.StdIn.readDouble() : Double
        var montante = scala.io.StdIn.readDouble() : Double
        var resultado = calculabonus(salario, montante) : Double
        printf("TOTAL = R$ %.2f\n", resultado);
    }
}

File: uri/bonus/bonus.go
package main

import (
	"C"
	"fmt"
)

func bonus(salario float64, montante float64) float64 {
	return (salario + ((montante * 15) / 100))
}

func main() {
	var nome string
	var salario, montante float64

	fmt.Scan(&nome)
	fmt.Scan(&salario)
	fmt.Scan(&montante)
	fmt.Printf("TOTAL = R$ %.2f\n", bonus(salario, montante))
}

/*
func main() {
	var salario, montante float64
	reader := bufio.NewReader(os.Stdin)
	nome, _ := reader.ReadString('\n')
	fmt.Scanf("%f\n", &salario)
	fmt.Scanf("%f\n", &montante)
	fmt.Printf("TOTAL = R$ %.2f", bonus(salario, montante))
}

*/
